<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogicalRuleView</class>
 <widget class="QWidget" name="LogicalRuleView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="ruleTitleLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Logical Diagram Rule&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QScrollArea" name="givensScroll">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="givensList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>269</width>
        <height>325</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="addGivenButton">
         <property name="text">
          <string>Add Given</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QScrollArea" name="resultsScroll">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="resultsList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>269</width>
        <height>325</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="addResultButton">
         <property name="text">
          <string>Add Result</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Results</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Givens</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
